Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -417,8 +422,9 @@ static const unsigned StringDataIs8BitFlag = 0x80000000;
  *    | ImageBitmapTag <originClean:uint8_t> <logicalWidth:int32_t> <logicalHeight:int32_t> <resolutionScale:double> DestinationColorSpace <byteLength:uint32_t>(<imageByteData:uint8_t>)
  *    | OffscreenCanvasTransferTag <value:uint32_t>
  *    | WasmMemoryTag <value:uint32_t>
- *    | RTCDataChannelTransferTag <processIdentifier:uint64_t><rtcDataChannelIdentifier:uint64_t><label:String>
+ *    | RTCDataChannelTransferTag <identifier:uint32_t>
```

Comment:
If this change is intentional, you should probably describe it in the commit log.

Reply:
Will do.